libcxx/test/std/thread/thread.threads/thread.thread.class/thread.thread.constr/F.verify.cpp

```diff
@@ -0,0 +1,94 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: no-threads
+// UNSUPPORTED: c++03
```

Comment:
Perhaps it's better to use the new style in this test file (for consistency), although it's longer.
```suggestion
// REQUIRES: std-at-least-c++11
```